src/main/kotlin/no/nav/syfo/ApplicationEnvironment.kt

```diff
@@ -20,8 +22,17 @@ data class Environment(
             ClientEnvironment(
                 baseUrl = getEnvVar("ISTILGANGSKONTROLL_URL"),
                 clientId = getEnvVar("ISTILGANGSKONTROLL_CLIENT_ID")
-            )
-        )
+            ),
+            pdl = ClientEnvironment(
+                baseUrl = getEnvVar("PDL_URL"),
```

Comment:
Disse må være definert i nais-filene og så må vi be pdl om tilgang for denne appen.

Reply:
Hvordan ber vi pdl om tilgang? Legger opp en PR på inbound policyen deres?

Reply:
Tror man kan be om tilgang til dev i #pdl på slack og så må man bestille i porten for prod: https://jira.adeo.no/plugins/servlet/desk/portal/541/create/3842